scripts/deploy.sh

```diff
@@ -50,7 +50,7 @@ redis_process=$!
 
 # Start the API
 cd /usr/src/app/api || exit 1
-uvicorn src.serge.main:app --host 0.0.0.0 --port 8008 || {
+hypercorn src.serge.main:app --bind 0.0.0.0:8008 --bind [::]:8008 || {
```

Comment:
This is forcing ipv6 to everyone. It needs to be an env option.

`SERGE_ENABLE_IPV6`